program-runtime/src/invoke_context.rs

```diff
@@ -196,7 +196,9 @@ pub struct InvokeContext<'a> {
     /// Runtime configurations used to provision the invocation environment.
     pub environment_config: EnvironmentConfig<'a>,
     /// The compute budget for the current invocation.
-    compute_budget: ComputeBudget,
+    compute_budget: SVMTransactionExecutionBudget,
+    /// The compute cost for the current invocation.
+    compute_cost: SVMTransactionExecutionCost,
```

Comment:
rename to `execution_cost` (same with accessor fn) to be consistent with the type name?